plugin.video.svtplay/addon.xml

```diff
@@ -0,0 +1,30 @@
+<addon id="plugin.video.svtplay" name="SVT Play" provider-name="nilzen" version="5.0.1">
+  <requires>
+    <import addon="script.module.requests" version="2.22.0" />
+    <import addon="xbmc.python" version="2.25.0" />
+  </requires>
+  <extension library="default.py" point="xbmc.python.pluginsource">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <language>sv</language>
+    <summary lang="en_GB">Watch content from SVT Play.</summary>
+    <summary lang="sv_SE">Titta på innehåll från SVT Play.</summary>
+    <description lang="en_GB">With this addon you can stream content from SVT Play (svtplay.se).</description>
+    <description lang="sv_SE">Med denna addon kan du strömma innehåll från SVT Play (svtplay.se).</description>
+    <disclaimer lang="en_GB">This is an unofficial addon and it is not supported or maintained by SVT. Some parts of this addon may not be legal in your country of residence - please check with your local laws before installing.</disclaimer>
+    <platform>all</platform>
+    <license> GNU GENERAL PUBLIC LICENSE. Version 3, 29 June 2007</license>
+    <website>https://github.com/nilzen/xbmc-svtplay</website>
```

Comment:
point to the content providers website

Reply:
That's not how I interpret the documentation: https://kodi.wiki/view/Addon.xml#.3Cwebsite.3E
"""
The <website> element provides the website URL for this specific add-on.
<website>http://www.myaddonwebsite.com/</website>
"""

Reply:
Thanks for pointing out the wiki could be improved. We'll have that changed. Website refers to the content website or in some cases to the add-on website (eg for any addon that provides content from multiple websites or that have a website dedicated to the add-on). Usually the source element already points to the upstream code so there's no need to repeat it. Can you have this changed for the PRs to be merged? Thanks

Reply:
I have changed this now.